tests/conftest.py

```diff
@@ -35,7 +35,7 @@
 if TYPE_CHECKING:
     from _pytest.python import Metafunc
 
-GH_MATRIX_LENGTH = 36
+GH_MATRIX_LENGTH = 33
```

Comment:
The matrix length for the `test_type_current` unit test in test_type.py seems to be different that the other ones, and due to that the conftest fixture for matrix_length doesn't match with it. Please suggest on how to tackle this. Should we hardcode the length just for this test instead of pulling from the fixture?

Reply:
Change looks legit but tests needs to be updated too. You are also welcomed to make them better if you don't like the "opaque numbers" there ;)

Reply:
I think it's setup correctly. I was just asking if it's okay to hard code the matrix length for this test case, instead of pulling from the fixture defined in conftest. Looks like there's probably no better way. So, I'll push that change.